package.json

```diff
@@ -234,7 +234,7 @@
           "when": "never"
         },
         {
-          "command": "azure-api-center.exportOpenApi",
+          "command": "azure-api-center.exportApi",
           "when": "never"
```

Comment:
For newly-added commands. let's support triggering it from commandPalette

Reply:
so we need to also support command palette to trigger the exportApi and let user select the node?